scanners/zap-advanced/templates/zap-advanced-scan-type.yaml

```diff
@@ -95,15 +95,17 @@ spec:
                 - "database.recoverylog=false" # Tune ZAP, DB recovery is not needed here
                 - "-config"
                 - "connection.timeoutInSecs=120" # Tune ZAP timeout by default to be 2minutes
-                {{ if and .Values.zapConfiguration.global .Values.zapConfiguration.global.addonUpdate }}
+                {{ if .Values.zapConfiguration.global}}
+                {{ if .Values.zapConfiguration.global.addonUpdate }}
                 - "-addonupdate"    # Enable AddOn Updates on startup if possible
                 {{- end }}
```

Comment:
I think this is missing an additional {{ end }} here?
This PR adds two new if but only one end 🤔

Reply:
Ah no sorry I've missread the diff, it's working as intended 👍